aea/cli/push.py

```diff
@@ -33,14 +34,19 @@ def push(ctx: Context, registry):
     """Push item to Registry or save it in local packages."""
     ctx.set_config("registry", registry)
 
+    packages_path = get_default_registry_path(ctx)
```

Comment:
Not a fan of this. The path is already available on `ctx` via `ctx.agent_config.registry_path` so no need to set it here, just pass `ctx` as an argument